editor/project_settings_editor.cpp

```diff
@@ -275,10 +275,8 @@ void ProjectSettingsEditor::_editor_restart_close() {
 void ProjectSettingsEditor::_action_added(const String &p_name) {
 	String name = "input/" + p_name;
 
-	if (ProjectSettings::get_singleton()->has_setting(name)) {
-		action_map->show_message(vformat(TTR("An action with the name '%s' already exists."), name));
-		return;
-	}
+	ERR_FAIL_COND_MSG(ProjectSettings::get_singleton()->has_setting(name),
+			"An action with the name already exists.");
```

Comment:
```suggestion
			"An action with this name already exists.");
```
Assuming that the name is visible in the UI so redundant in the message.

Same for the other message.

Reply:
Done.